hoot-core/src/main/cpp/hoot/core/conflate/highway/RemoveRoundabouts.cpp

```diff
@@ -50,7 +50,6 @@ HOOT_FACTORY_REGISTER(OsmMapOperation, RemoveRoundabouts)
 
 RemoveRoundabouts::RemoveRoundabouts()
 {
-  // blank
```

Comment:
@bwitham these comments were in here because the Qube didn't like empty functions that didn't have a comment in it.  I never liked that rule and we should turn it off.  I'll add an exception to the rule and send it to you so you can commit it with your other change from the code review.  I noticed that the Qube didn't run.  @sputnam did we stop running the Qube?

Reply:
ok, will add it.  thanks.